test-utils/global-state-manager.ts

```
@@ -0,0 +1,79 @@
+  }
+
+  static getServiceOfferings() {
+    const state = this.loadState();
+    return state.serviceOfferings;
+  }
+
+  static getSubscriptions() {
+    const state = this.loadState();
+    return state.subscriptions;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)